android/pytorch_android/src/main/java/com/facebook/pytorch/PytorchScriptModule.java

```diff
@@ -0,0 +1,34 @@
+package com.facebook.pytorch;
+
+import com.facebook.jni.HybridData;
+
+import java.nio.Buffer;
+
+public class PytorchScriptModule {
+
+  private NativePeer mNativePeer;
+
+  public PytorchScriptModule(final String modelAbsolutePath) {
+    this.mNativePeer = new NativePeer(modelAbsolutePath);
+  }
+
+  public Tensor run(Tensor input) {
```

Comment:
It should receive a list of IValues and return a single IValue

Reply:
Yes :) I added PR as preview for IValue support
https://github.com/pytorch/pytorch/pull/24363